packages/block-library/src/navigation/edit/navigation-menu-delete-control.js

```diff
@@ -1,10 +1,7 @@
 /**
  * WordPress dependencies
  */
-import {
-	Button,
-	__experimentalConfirmDialog as ConfirmDialog,
-} from '@wordpress/components';
+import { Button, ConfirmDialog } from '@wordpress/components';
```

Comment:
I think updating all of Gutenberg to use the stable component imports can be its own PR. There are quite a few of them that this PR isn't covering just yet, and splitting to smaller PRs could help with reviewing.

Reply:
Doing it in a separate PR would also help to check that we didn't remove any `__experimental` exports by mistake.

Reply:
Sounds good! I'll rework that 👍🏻